src/internal.c

```diff
@@ -12189,21 +12189,13 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                             "\tCallback override available, will continue");
                         /* check if fatal error */
                         args->fatal = (args->verifyErr) ? 1 : 0;
-                    #if defined(WOLFSSL_EXTRA_ALERTS) || \
```

Comment:
What's the point of gating this out based on these macros? Otherwise, we don't send an alert when cert verification fails, and the client won't know why the connection ended. It seems like that's something we would want to always do, not just in certain configurations. I've taken out the gates accordingly, but if there's a good reason to have them, I'll add them back.

Reply:
That is a good question. Historically there are some cases where sending an alert could leak information about the padding around a key, however I do not believe this is one of them.

Reply:
The RFC leaves a lot of the alerts ambiguous and optional. I can't find any reason for this gate apart from backwards compatibility. I think it may be worth to leave the macro's as they are in case a user doesn't want to send alerts.